config.go

```diff
@@ -15,4 +15,5 @@ type clientConfig struct {
 	Store             loopdb.SwapStore
 	LsatStore         lsat.Store
 	CreateExpiryTimer func(expiry time.Duration) <-chan time.Time
+	LoopOutMaxParts   uint32
```

Comment:
im not sure if this is the right place to put this. Let me know if there is a better way to give the swapclient_server access to this value